CrawlerTests/AnalyzersTests/PunctuationAnalyzerTests.cs

```diff
@@ -7,33 +7,58 @@
 using Moq;
 using System.Collections.Generic;
 using System.Linq;
+using Crawler.PartOfSpeechTagger;
 using Xunit;
 
 namespace CrawlerTests.AnalyzersTests
 {
     public class PunctuationAnalyzerTests
     {
-        private readonly ILexer lexer;
-        private readonly LexerConfig config;
-        private readonly PunctuationAnalyzer punctuationsAnalyzer = new PunctuationAnalyzer();
+	    private const string PASSIVE_VOICE_SENTENCE = "The house will be cleaned by me every Saturday.";
+	    private const string ACTIVE_VOICE_SENTENCE = " Sue changed the flat tire. ";
+
+	    private ILexer lexer;
```

Comment:
something is weird with the indentation

Reply:
again probably a problem with git:
![image](https://user-images.githubusercontent.com/21021560/74935188-84978380-53f0-11ea-8208-d2567fdb3822.png)